pkg/controllers/securitypolicy_associated_controller.go

```diff
@@ -0,0 +1,241 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
```

Comment:
securitypolicy_associated_controller.go, seemly is not appropriate name for this file, since this file is focusing on named port event filtering and listing. Also there is no a new controller implemented in this file. Is it ok to change fine name to  like securitypolicy_namedport_handler.go

Reply:
I also considered it previously,  the reason why I choose this name is that it focuses on `associated` meaning, not just for named port feature, it is extensible.